docs/telemetry.md

```diff
@@ -1,99 +1,193 @@
 # Telemetry
 
-Graphistry services can be configured to export telemetry information using the [OpenTelemetry](https://opentelemetry.io/) standard. This includes logs, metrics, and traces.
+## Overview
 
-When enabled, telemetry sent to the local Graphistry [OTEL Collector](https://opentelemetry.io/docs/collector/) service, and from there, dispatches to any observability tool that is compatible with the OpenTelemetry standard (e.g. Prometheus, Jaeger, Grafana Cloud, etc.).
+Graphistry services export telemetry information (metrics and traces) using the [OpenTelemetry](https://opentelemetry.io/) standard.
 
-To use telemetry with Graphistry, you need to:
+Graphistry services push their telemetry data to the [opentelemetry-collector](https://opentelemetry.io/docs/collector/) service (alias `otel-collector`) and this will forward the data to any observability tool that is compatible with the OpenTelemetry standard (e.g. Prometheus, Jaeger, Grafana Cloud, etc.).
 
-* Enable telemetry collection recording in one or more Graphistry services
-* Configure which users each service will monitor (see section on feature flags)
-* Enable the telemetry collector service
-* Configure the forwarding from the telemetry collector to an observability database/UI
-* Optionally, setup a local observability database/UI
+Graphistry can be deployed alongside the packaged Graphistry Local Telemetry Suite, which includes Prometheus, Jaeger, and Grafana services.
+
+## Usage
```

Comment:
A bit confusing to me is we support roughly 3 scenarios, but I can't distinguish these in the docs:

* Kubernetes: NOT these docs

* Forwarding: User hosts their own external otel services (Grafana Cloud, ...), so just about configuring forwarding

* Kitchen sink:  Configuration here is both for forwarding + managing graphistry-defined local otel dashboards etc

Reply:
Good point, I think it may be a good idea to add more about those different modes.